priv/repo/migrations/20160906195240_add_location_id_to_runs.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Gatherto.Repo.Migrations.AddLocationIdToRuns do
+  use Ecto.Migration
+
+  def change do
+    alter table(:runs) do
+      add :location_id, :string
+    end
+  end
+end
```

Comment:
I opted to keep it simple and just add a location_id to runs since right now the run model would just be 2 ids - a local id and a Google place id. If this changes in the future we can always migrate these to a new model.